File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/PromotionResponse.java
package com.cinema.ticket_booking.data.model.response;

import com.google.gson.annotations.SerializedName;

public class PromotionResponse {
    @SerializedName("id")
    private String id;

    @SerializedName("title")
    private String title;

    @SerializedName("description")
    private String description;

    @SerializedName("imageUrl")
    private String imageUrl;

    @SerializedName("targetUrl")
    private String targetUrl;

    public String getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public String getTargetUrl() {
        return targetUrl;
    }
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/PromotionResponse.kt
package com.cinema.ticket_booking.data.model.response

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class PromotionResponse(
    @SerializedName("id")
    val id: String?,

    @SerializedName("title")
    val title: String?,

    @SerializedName("description")
    val description: String?,

    @SerializedName("imageUrl")
    val imageUrl: String?,

    @SerializedName("targetUrl")
    val targetUrl: String?,

    @SerializedName("startDate")
    val startDate: String?,

    @SerializedName("endDate")
    val endDate: String?
) : Serializable
